src/etc/inc/plugins.inc.d/ipsec.inc

```diff
@@ -542,6 +544,22 @@ function ipsec_mobilekey_sort()
     });
 }
 
+function ipsec_keypairs()
```

Comment:
it looks like `ipsec_keypairs()` is only used in src/www/vpn_ipsec_phase1.php, in which case I propose to move it to there.

Reply:
Updated, this function now resides in `src/www/vpn_ipsec_phase1.php` as proposed.